versions/0.0.0.md

```diff
@@ -191,7 +191,7 @@ Requirements
    its value SHOULD be the user name of an existing EASY user account.
 
 #### 1.3 Manifests
-1. (a) The bag MUST have a SHA-1 payload manifest. (b) It MUST have entries for all the payload files.
+1. **(AIP)** (a) The bag MUST have a SHA-1 payload manifest. (b) It MUST have entries for all the payload files.
```

Comment:
```suggestion
1. **(SIP)** (a) The bag MUST have a payload manifest. (b) It MUST have entries for all the payload files.
**(AIP)** (a) The bag MUST have a SHA-1 payload manifest. (b) It MUST have entries for all the payload files.
```

Reply:
SIPs also need a payload manifest. Should we enumerate, or give examples?

Reply:
The requirement that there is at least *some* payload manifest is covered by the requirement that the bag is valid.

Reply:
We *could* consider requiring a minimum strength of SHA-1 for the SIP payloadmanifest.